TF_Service_Utilities/Installer/TouchFree_Installer.iss

```diff
@@ -4,7 +4,7 @@
 #define CompanyURL "https://ultraleap.com"
 #define ProductName "TouchFree"
 #define Publisher "Ultraleap Inc."
-#define ReleaseVersion "1.0.0-alpha5"
+#define ReleaseVersion "2.0.0-beta1"
```

Comment:
We could consider replacing this via the same method we replace the entries in the version.txt so that we don't have to replace them each time we update